specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/ContainerApps.json

```
@@ -628,10 +628,11 @@
           ]
         },
         "activeRevisionsMode": {
-          "description": "ActiveRevisionsMode controls how active revisions are handled for the Container app:\n<list><item>Multiple: multiple revisions can be active.</item><item>Single: Only one revision can be active at a time. Revision weights can not be used in this mode. If no value if provided, this is the default.</item></list>",
+          "description": "ActiveRevisionsMode controls how active revisions are handled for the Container app:\n<list><item>Multiple: multiple revisions can be active.</item><item>Single: Only one revision can be active at a time. Revision weights can not be used in this mode. If no value if provided, this is the default.</item><item>Labeled: Only the revisions that have labels attached to them are active.</item></list>",
           "enum": [
             "Multiple",
-            "Single"
+            "Single",
+            "Labeled"
```

```suggestion
            "Labels"
```

Labels would be more consistent with the APIs like `/{containerAppName}/labels`